torch/csrc/jit/import.cpp

```diff
@@ -112,6 +114,26 @@ void ScriptModuleDeserializer::deserialize(
   device_ = device;
 
   const auto& module_def = model_def.main_module();
+
+  // Load extra files.
+  for (const auto& kv : extra_files) {
```

Comment:
This would be as simple as this if we removed the entry in model_def:
```
reader_.getRecord("extra/" + kv.first)
```

Reply:
I guess we store the size to make it mmapable. Though for extra files it might not be necessary (and probably can be extracted from zip format anyway)